src/test/java/org/elasticsearch/test/rest/ElasticsearchRestTestCase.java

```diff
@@ -46,22 +54,44 @@
 import org.junit.Test;
 
 import java.io.IOException;
-import java.nio.file.FileSystems;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 import java.nio.file.Path;
 import java.nio.file.PathMatcher;
-import java.nio.file.Paths;
-import java.util.*;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
 /**
  * Runs the clients test suite against an elasticsearch cluster.
  */
-//tests distribution disabled for now since it causes reporting problems,
-// due to the non unique suite name
-//@ReplicateOnEachVm
-@AbstractRandomizedTest.Rest
+@ElasticsearchRestTestCase.Rest
+@Slow
```

Comment:
I see that this is slow and I know why you did this. I was hoping that distributing it to the different jvms would help though. This test is very important given that it's the only one that covers the rest layer. It takes time just because it contains all of the tests for the REST layer (think of a test class that holds ~450 tests). Are we sure we want to mark it @Slow and not execute it by default? I tend to be -1 on this.